packages/python-freesimplegui/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <aur at chrisarndt dot de>
+
+_projectname=FreeSimpleGUI
+_name=${_projectname,,}
+pkgname=python-$_name
+pkgver=5.2.0.post1
```

Comment:
Apparently this is identical to 5.2.0, but do I understand it correctly that the added `pyproject.toml` is kind of important? This is just a weird version string to use :D